hatchet/readers/caliper_native_reader.py

```diff
@@ -87,7 +87,11 @@ def _create_metric_df(self, metrics):
             if self.filename_or_caliperreader.attribute(col).is_value():
                 self.metric_cols.append(col)
         df_metrics = pd.DataFrame.from_dict(data=metrics)
-        df_new = df_metrics.groupby(df_metrics["nid"]).aggregate("first").reset_index()
+        # Aggregate on nid if timeseries data
+        if "timeseries.snapshot" in df_metrics:
```

Comment:
@slabasan Need a better way to identify a timeseries profile. I checked metadata, not guaranteed to have information in there, e.g.:
```
hatchet/tests/data/caliper-timeseries-cali/timeseries.cali
{'cali.caliper.version': '2.10.0-dev', 'cali.channel': 'default'}
```
So I'm using whether the `timeseries.snapshot` column is present. However this does not work for the
```
thicket/tests/data/example-timeseries/mem_power_timeseries.cali
```
example which does not have any `timeseries.*` columns.